dev/src/Command/NewComponentCommand.php

```diff
@@ -88,6 +89,13 @@ protected function configure()
                 null,
                 InputOption::VALUE_NONE,
                 'Do not run the update-component command after adding the component skeleton'
+            )
+            ->addOption(
+                'timeout',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'The timeout limit for executing commands in seconds. Defaults to 60.',
+                60
```

Comment:
do we want to bump this up by default, like `120`, since we think this will generally take longer?

Reply:
Only chose `60` as is the default for Symfony soooo no strong feelings. Will change it.